addheader.cfg

```diff
@@ -0,0 +1,8 @@
+---
+root: idaes
+text: COPYRIGHT.md
+patterns:
+   - "*.py"
+   - "~__init__.py"
```

Comment:
Does the leading tilde in the pattern mean something special? If so, it could be useful to add a comment here for quick reference

Reply:
yeah it's in the docs but who reads those so will add comment